conda/cli/main_env_remove.py

```diff
@@ -52,34 +54,28 @@ def configure_parser(sub_parsers: _SubParsersAction, **kwargs) -> ArgumentParser
     add_parser_solver(p)
     add_output_and_prompt_options(p)
 
-    p.set_defaults(func="conda.cli.main_env_remove.execute")
+    p.set_defaults(
+        func="conda.cli.main_remove.execute",
+        all=True,
+        channel=None,
+        features=None,
+        override_channels=None,
+        use_local=None,
+        use_cache=None,
+        offline=None,
+        package_names=[],
+        force=True,
+        pinned=None,
+        keep_env=False,
+    )
 
     return p
 
 
+@deprecated("25.3", "99.9", addendum="Use `conda.cli.main_remove.execute` instead.")
```

Comment:
Oh we want an actual removal version here though since this execute function no longer serves a purpose.

```suggestion
@deprecated("25.3", "25.9", addendum="Use `conda.cli.main_remove.execute` instead.")
```